spharpy/spherical.py

```diff
@@ -248,6 +248,108 @@ def n3d_to_sn3d_norm(n):
     return 1 / np.sqrt(2 * n + 1)
 
 
+def renormalize(data, channel_convention, current_norm, target_norm, axis):
+    """
+    Renormalize either an array of spherical harmonics coefficients or basis
+    functions.
+
+    Parameters
+    ----------
+    data : ndarray
+        Data which should be renormalized, either spherical harmonics
+        coefficients or basis functions.
+    channel_convention : str
+        Channel convention of the data which should be renormalized. Valid
+        conventions are `"acn"` or `"fuma"`.
+    current_norm : str
+        Current normalization. Valid normalizations are `"n3d"`, `"maxN"`, or
+        `"sn3d"`.
+    target_norm : str
+        Desired normalization. Valid normalizations are `"n3d"`,
+        `"maxN"`, or `"sn3d"`.
+    axis : integer
+        Axis along which the renormalization should be applied
+
+    Returns
+    -------
+    data : ndarray
+        Renormalized data
+    """
+    if target_norm not in ["n3d", "maxN", "sn3d"]:
```

Comment:
I would suggest to make this check also for `current_norm` and `channel_convention`